redash/query_runner/mysql.py

```diff
@@ -46,9 +46,25 @@ def configuration_schema(cls):
                     'type': 'string',
                     'title': 'Database name'
                 },
-                "port": {
-                    "type": "number"
+                'port': {
+                    'type': 'number'
                 },
+                'use_ssl': {
+                    'type': 'boolean',
+                    'title': 'Use SSL'
+                },
+                'ssl_cacert': {
+                    'type': 'string',
+                    'title': 'CA certificate to verify peer against (SSL)'
+                },
+                'ssl_cert': {
+                    'type': 'string',
+                    'title': 'Client certificate file (SSL)'
+                },
+                'ssl_key': {
+                    'type': 'string',
+                    'title': 'Private key filename (SSL)'
+                }
             },
```

Comment:
Let's just normalize the titles to:

ssl_cacert: "Path to CA certificate file to verify peer against (SSL)"
ssl_cert: "Path to client certificate file (SSL)"
ssl_key: "Path to private key file (SSL)"